node-operators/indexers/thegraph-node.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Graph Node
+description: Build APIs using The Graph indexing protocol on Moonbeam
+---
+
+# Running a Graph Node on Moonbeam
+
+![The Graph Node on Moonbeam](/images/thegraph/thegraphnode-banner.png)
+
+## Introduction
+
+A Graph Node sources events from a blockchain to deterministically update a data store that can be then queried via a GraphQL endpoint.
+
+There are two ways you can set up a Graph Node: you can use Docker to run an all-in-one image, or you can run their [Rust implementation](https://github.com/graphprotocol/graph-node). The steps described in this guide will only cover the Docker alternative, as it is more convenient, and you can set up a Graph Node very quickly.
+
+!!! note
+    The steps described in this guide assume an Ubuntu 18.04-based environment, and they will need to be adapted accordingly for other systems.
```

Comment:
oh just noticed this... I ran through these steps on MacOS and it works! So you can use the text snippet for this